cpp/src/sampling/random_walks.cuh

```diff
@@ -1003,18 +1031,27 @@ struct coo_convertor_t {
  * @brief returns random walks (RW) from starting sources, where each path is of given maximum
  * length. Uniform distribution is assumed for the random engine.
  *
- * @tparam graph_t Type of graph (view).
+ * @tparam graph_t Type of graph/view (typically, graph_view_t).
  * @tparam index_t Type used to store indexing and sizes.
  * @param handle RAFT handle object to encapsulate resources (e.g. CUDA stream, communicator, and
  * handles to various CUDA libraries) to run graph algorithms.
- * @param graph Graph object to generate RW on.
+ * @param graph Graph (view )object to generate RW on.
  * @param ptr_d_start Device pointer to set of starting vertex indices for the RW.
  * @param num_paths = number(paths).
  * @param max_depth maximum length of RWs.
+ * @param use_padding (optional) specifies if return uses padded format (true), or coalesced
+ * (compressed) format; when padding is used the output is a matrix of vertex paths and a matrix of
+ * edges paths (weights); in this case the matrices are stored in row major order; the vertex path
+ * matrix is padded with `num_vertices` values and the weight matrix is padded with `0` values;
  * @return std::tuple<device_vec_t<vertex_t>, device_vec_t<weight_t>,
- * device_vec_t<index_t>> Triplet of coalesced RW paths, with corresponding edge weights for
- * each, and coresponding path sizes. This is meant to minimize the number of DF's to be passed to
- * the Python layer.
+ * device_vec_t<index_t>> Triplet of either padded or coalesced RW paths; in the coalesced case
```

Comment:
What is device_vec_t? I guess this alias is removed.

Reply:
`device_vec_t` is an alias that has been there already. I have explained the motivation of its existence in the very 1st PR that had that code.

Reply:
Oh, my bad. I see, that's in the public API, I'll fix that.

Reply:
Done.